ckan/logic/action/update.py

```diff
@@ -1084,16 +1084,15 @@ def _bulk_update_dataset(context, data_dict, update_dict):
     org_id = data_dict.get('org_id')
 
     model = context['model']
-
     model.Session.query(model.package_table) \
         .filter(model.Package.id.in_(datasets)) \
         .filter(model.Package.owner_org == org_id) \
         .update(update_dict, synchronize_session=False)
 
     # revisions
-    model.Session.query(model.package_table) \
-        .filter(model.Package.id.in_(datasets)) \
-        .filter(model.Package.owner_org == org_id) \
+    model.Session.query(model.package_revision_table) \
+        .filter(model.PackageRevision.id.in_(datasets)) \
+        .filter(model.PackageRevision.owner_org == org_id) \
         .update(update_dict, synchronize_session=False)
```

Comment:
pretty sure we need to restrict to the latest revision here, don't we?

This code was changed in https://github.com/ckan/ckan/commit/0413e259ce2c80925c5aadca0896f264122567e4#diff-4b95e302501a681a72c099ecc904de3dL1293 there should be a `current=True` in this query